pkg/domain/infra/abi/play.go

```diff
@@ -267,12 +267,9 @@ func (ic *ContainerEngine) playKubePod(ctx context.Context, podName string, podY
 	}
 
 	if podOpt.Infra {
-		imagePull := config.DefaultInfraImage
```

Comment:
What happens if the user specified an infra image?

Reply:
If users specified an infra image, it will be used in `containerConfig := util.DefaultContainerConfig()`.

This logic is already exist in `containers/common/pkg/config` package.

Reply:
I will add an e2e test later.